sdk/schemaregistry/Azure.Data.SchemaRegistry/src/Generated/SchemaRegistryClient.cs

```diff
@@ -0,0 +1,872 @@
+        /// <exception cref="ArgumentException"> <paramref name="groupName"/> or <paramref name="name"/> is an empty string, and was expected to be non-empty. </exception>
+        /// <exception cref="RequestFailedException"> Service returned a non-success status code. </exception>
+        /// <returns> The response returned from the service. </returns>
+        /// <include file="Docs/SchemaRegistryClient.xml" path="doc/members/member[@name='GetSchemaByVersion(string,string,int,RequestContext)']/*" />
+        public virtual Response GetSchemaByVersion(string groupName, string name, int schemaVersion, RequestContext context)
+        {
+            Argument.AssertNotNullOrEmpty(groupName, nameof(groupName));
+            Argument.AssertNotNullOrEmpty(name, nameof(name));
+
+            using var scope = ClientDiagnostics.CreateScope("SchemaRegistryClient.GetSchemaByVersion");
+            scope.Start();
+            try
+            {
+                using HttpMessage message = CreateGetSchemaByVersionRequest(groupName, name, schemaVersion, context);
+                return _pipeline.ProcessMessage(message, context);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary> Get ID for existing schema. </summary>
+        /// <param name="groupName"> Name of schema group. </param>
+        /// <param name="name"> Name of schema. </param>
+        /// <param name="schemaContent"> String representation (UTF-8) of the registered schema. </param>
+        /// <param name="contentType"> The content type for given schema. Allowed values: "application/json; serialization=Avro" | "application/json; serialization=json" | "text/plain; charset=utf-8" | "text/vnd.ms.protobuf". </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="groupName"/>, <paramref name="name"/> or <paramref name="schemaContent"/> is null. </exception>
+        /// <exception cref="ArgumentException"> <paramref name="groupName"/> or <paramref name="name"/> is an empty string, and was expected to be non-empty. </exception>
+        /// <remarks> Gets the ID referencing an existing schema within the specified schema group, as matched by schema content comparison. </remarks>
+        internal virtual async Task<Response> GetSchemaIdByContentAsync(string groupName, string name, BinaryData schemaContent, Core.ContentType contentType, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(groupName, nameof(groupName));
+            Argument.AssertNotNullOrEmpty(name, nameof(name));
+            Argument.AssertNotNull(schemaContent, nameof(schemaContent));
+
+            RequestContext context = FromCancellationToken(cancellationToken);
+            using RequestContent content = RequestContent.Create(schemaContent);
```

Comment:
I had to change this from
```csharp
using RequestContent context = RequestContentHelper.FromObject(schemaContent);
```
which uses a JsonSerializer. The JsonSerializer doesn't work with custom schemas and also causes some unexpected behavior with Json schemas sometimes